cairo-sys-rs/src/lib.rs

```diff
@@ -275,6 +275,11 @@ pub type cairo_read_func_t =
 pub type cairo_write_func_t =
     Option<unsafe extern "C" fn(*mut c_void, *mut c_uchar, c_uint) -> cairo_status_t>;
 
+#[cfg(any(feature = "freetype", feature = "dox"))]
+pub type FT_Face = *mut c_void;
+#[cfg(any(feature = "freetype", feature = "dox"))]
+pub type FcPattern = c_void;
```

Comment:
Why is one a pointer and the other not, but always used as a pointer below?

Reply:
This is to match how the functions are defined in cairo - `FT_Face` is defined in freetype to be a pointer, but `FcPattern` in fontconfig is defined as an object. See https://cairographics.org/manual/cairo-FreeType-Fonts.html#cairo-ft-font-face-create-for-ft-face.

Reply:
We can define them differently in rust if you think it makes sense to.

Reply:
No, that explanation makes sense